app/routes/_auth+/login.tsx

```diff
@@ -40,9 +40,11 @@ import {
 } from "~/utils/http.server";
 import { validEmail } from "~/utils/misc";
 
-export function loader({ context }: LoaderFunctionArgs) {
+export async function loader({ context, request }: LoaderFunctionArgs) {
+  let lng = getLng(request);
```

Comment:
So every time we need translations in a loader we need to call those 2:
```ts
let lng = getLng(request);
let t = await i18next.getFixedT(lng);
```

I think we should simplify that. Like in the components we have a hook, for loaders/actions we need to have a function that receives request and returns `t()` without having to do this manually every time. Also another important point, does this consume the request? Its important to know so we can then know if we should clone it before using it.